src/Utilities/functions.jl

```diff
@@ -1296,3 +1296,51 @@ function operate(::typeof(vcat), ::Type{T},
     fill_vector(constant, T, 0, 0, fill_constant, output_dim, funcs...)
     return VAF(terms, constant)
 end
+
+function scalarize(f::MOI.VectorOfVariables)
```

Comment:
What is the difference with eachscalar?

Reply:
not sure about this point.
scalarize is no very complex, and I am not sure about the performance difference.
Do we plan to solve #418 in the short time?
Since this bridge could be used in the short time I would keep a more efficient method for a bit.

Reply:
That's fine, leave a comment before scalarize to mention this